pkgs/by-name/or/orca-slicer/package.nix

```diff
@@ -15,6 +16,8 @@ bambu-studio.overrideAttrs (
       hash = "sha256-h+cHWhrp894KEbb3ic2N4fNTn13WlOSYoMsaof0RvRI=";
     };
 
+    nativeBuildInputs = previousAttrs.nativeBuildInputs ++ [ gcc13 ];
```

Comment:
I'm not sure what the implications of this are. I think it might be cleaner to just override the stdenv with `gcc13Stdenv`.